### PR TITLE
Refactor sentinel message into shared place.

### PR DESCRIPTION
**Pull request by tz70s**
Previously, kubernetes container depends on sentinel message defined by
docker container object. Not ideal for separated SPI impl, put this to a
share place instead.

Signed-off-by: Tzu-Chiao Yeh <su3g4284zo6y7@gmail.com>

## Related issue and scope

- [ ] I opened an issue to propose and discuss this change (#????)

## My changes affect the following components

- [ ] API
- [ ] Controller
- [ ] Message Bus (e.g., Kafka)
- [ ] Loadbalancer
- [x] Invoker
- [ ] Intrinsic actions (e.g., sequences, conductors)
- [ ] Data stores (e.g., CouchDB)
- [x] Tests
- [ ] Deployment
- [ ] CLI
- [ ] General tooling
- [ ] Documentation

## Types of changes

- [ ] Bug fix (generally a non-breaking change which closes an issue).
- [x] Enhancement or new feature (adds new functionality).
- [ ] Breaking change (a bug fix or enhancement which changes existing behavior).

## Checklist:

- [x] I signed an [Apache CLA](https://github.com/apache/incubator-openwhisk/blob/master/CONTRIBUTING.md).
- [x] I reviewed the [style guides](https://github.com/apache/incubator-openwhisk/wiki/Contributing:-Git-guidelines#code-readiness) and followed the recommendations (Travis CI will check :).
- [x] I added tests to cover my changes.
- [ ] My changes require further changes to the documentation.
- [ ] I updated the documentation where necessary.

### CODE REVIEW

**Comment by tz70s** [comment]
@rabbah @markusthoemmes  thanks; yes, it's quite redundant and helps nothing. I think some tests in `ActionContainer` and `WskBasicTests` can test this, but not sure the visibility is correct? I would like to make sure that we can catch up if the sentinel message get changed in some day.

**Comment by tz70s** [comment]
A question might be relevant, since I introduce the term `ContainerLog`. It's a little weird the `logging` package existed besides it. Can we rename it into `logstore`? Since it only contains `logStore` classes.

**Comment by rabbah** [comment]
Why not put it in the Container singleton.

**Comment by tz70s** [comment]
I think it's a trait or am I missing something?

https://github.com/apache/incubator-openwhisk/blob/24c6f3e3b1edf082ba720fa088a1ab26ba35c217/common/scala/src/main/scala/whisk/core/containerpool/Container.scala#L58-L65

**Comment by markusthoemmes** [comment]
You can give it an companion `object`. Just rename `ContainerLog` to `Container` and you'll be fine.

**Comment by tz70s** [comment]
Thanks! In addition, I also remove the redundant val modifier in case classes above.

File: common/scala/src/main/scala/whisk/core/containerpool/Container.scala [hunk 1]
```diff
@@ -48,13 +48,24 @@ import whisk.core.entity.ActivationEntityLimit
...
 }
-case class ContainerAddress(val host: String, val port: Int = 8080) {
+case class ContainerAddress(host: String, port: Int = 8080) {
   require(host.nonEmpty, "ContainerIp must not be empty")
 }
 
+object Container {
+
+  /**
+   * The action proxies insert this line in the logs at the end of each activation for stdout/stderr
```
Comment (rabbah) [inline common/scala/src/main/scala/whisk/core/containerpool/Container.scala]: small nit, can you end this sentence with a `.`
then squash all the commits to clean up the commit message.

**Pull request merged**